Projects/App/Sources/Generic/AppFlow.swift

```diff
@@ -48,14 +48,58 @@ struct EmbarkOnboardingFlow: Presentable {
 
         return (viewController, Future { completion in
             bag += storySignal.atValue { story in
+                let embark = Embark(name: story.name, flowType: .onboarding)
+                bag += embark.routeSignal.onValue { route in
```

Comment:
Maybe its cleaner if embark returns a `FiniteSignal<Either<Route, Redirect>>`?

Reply:
I didn't know this was a thing! CLEAN